interface/super/edit_list.php

```diff
@@ -1058,6 +1088,49 @@ function mysubmit() {
                     </select>
                 </div>
 
+                <!--Added filter-->
+                <script type="text/javascript">
+                    function lister() {
+                        var urlFull   = new URL($(location).attr('href'));
+                        var list_from = parseInt($("#list-from").val());
+                        var list_to   = parseInt($("#list-to").val());
+                        var list_id_container = $("#list_id").val();
+
+                        if( list_from > list_to ){
+                            alert(<?php echo xlj("Please enter a enter valid range"); ?>);
+                            return false;
+                        }
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_from");
+                            urlFull.searchParams.set("list_from",list_from);
+                        }
+
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_to");
+                            urlFull.searchParams.set("list_to",list_to);
+                        }
+                        if( list_id_container.length > 0 ){
+                            urlFull.searchParams.delete("list_id_container");
+                            urlFull.searchParams.set("list_id_container", list_id_container );
+                        }
+                        window.location.replace(urlFull);
+                    }
+                </script>
+                <?php
+                    $urlFrom = ( isset($_REQUEST["list_from"]) && intval($_REQUEST["list_from"]) >= 0 ? intval($_REQUEST["list_from"]) :  1 );
+                    $urlTo   = ( isset($_REQUEST["list_to"]) && intval($_REQUEST["list_to"]) >= 0 ? intval($_REQUEST["list_to"]) : '' );
+                    if( $urlTo == 0){
+                        $urlTo = '';
+                    }
+                ?>
+                <div class="blck-filter" style="float: left; margin-top: 5px; margin-left: 10px; border:0px solid red; width: auto; ">
+                    <div id="input-type-from" style="float: left; "><?php echo xlt("From"); ?>&nbsp;<input aumtocomplete="off" id="list-from" value="<?php echo attr($urlFrom);?>" style = "margin-right: 10px; width: 40px;">
+                        <?php echo xlt("To"); ?>&nbsp;<input autocomplete="off" id="list-to" value="<?php echo attr($urlTo); ?>" style=" margin-right: 10px; width: 40px;">
+                    </div>
+                    <div style="float:left" ><input type="button" value="<?php echo xlt('Show records'); ?>" onclick="lister()"></div>
```

Comment:
change the xlt() in above line to xla()

Reply:
OK

Reply:
I only meant to change in the 1 line above, not the others :)

Reply:
Which line?